apps/backend/__init__.py

```
@@ -48,24 +48,44 @@
-    logger.warning("agents.tools module not found, using empty list placeholder")
-    ALL_TOOLS = []
-
-# For backward compatibility with tests expecting flask_app
-app = fastapi_app  # Tests can use this
+# Lazy loading to avoid importing entire backend app on package import
+# This allows CLI tools and tests to import from apps.backend without
+# triggering the full FastAPI app initialization
+_lazy_imports = {}
+
+
+def __getattr__(name):
```

Ruff (0.14.4)

Missing return type annotation for private function `__getattr__`

(ANN202)